setup.py

```diff
@@ -308,6 +308,9 @@ def run_tests(self):
 
     install_requires=requirements,
     tests_require=test_requirements,
+    extras_require={
```

Comment:
Does setuptools 1.0 handle extras properly? (I don't know that I would say it needs to since it's not unreasonable to ask developers to be vaguely current, just curious)